src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -2430,6 +1307,110 @@ export default class Aircraft {
         }
     }
 
+    /**
+     * @for AircraftInstanceModel
+     * @method aircraftTurnPhysics
+     * This turns the aircraft if it is not on the ground and has not arived at its destenation
+     */
+    aircraftTurnPhysics() {
+        // Exits eary if the airplane is on the ground or at its destenation
+        if (this.isOnGround() && this.heading === this.target.heading) {
+            return;
+        }
+        // TURNING
+        // this.target.heading = radians_normalize(this.target.heading);
+        // Perform standard turns 3 deg/s or 25 deg bank, whichever
+        // requires less bank angle.
+        // Formula based on http://aviation.stackexchange.com/a/8013
+        const turn_rate = clamp(0, 1 / (this.speed / 8.883031), 0.0523598776);
+        const turn_amount = turn_rate * window.gameController.game_delta();
+        const offset = angle_offset(this.target.heading, this.heading);
+
+        if (abs(offset) < turn_amount) {
+            this.heading = this.target.heading;
+        } else if ((offset < 0 && this.target.turn === null) || this.target.turn === 'left') {
```

Comment:
is `null` a valid value for `this.target.turn`?  It could be, there are `null`s all over the place when there really shouldn't be (but that is a different issue).

In javascript, `null` is actually an object.  when doing comparisons like `this.target.turn === null` you are actually checking to see if that property equals the `null` object.  That may be different that what you are expecting.  If `this.target.turn` is initialized as `''`, that comparison won't work as expected here.

This is likely correct, but I wanted to call that out just in case.